pyomo/contrib/doe/fim_doe.py

```diff
@@ -934,16 +950,17 @@ def compute_FIM(self, design_values, mode='sequential_finite', FIM_store_name=No
 
             # set ub and lb to parameters
             for par in self.param_name:
-                component = eval('mod.'+par+'[0]')
+                #component = eval('mod.'+par+'[0]')
+                component = eval('mod.'+par)
```

Comment:
Is there a safer way to do this in Pyomo? What does parmest do?

Reply:
ComponentUID